Looped/Interface/Popular/PopularViewController.swift

```diff
@@ -6,15 +6,26 @@ final class PopularViewController: ViewController {
     
     // MARK: - Properties
     private let popularView = PopularView()
-    weak var flowDelegate: PopularViewControllerFlowDelegate?
+    var flowDelegate: PopularViewControllerFlowDelegate?
```

Comment:
to musi zostac `weak` - trzeba tylko znalezc rozwiazanie dlaczego tak sie dzieje